modules/local/cat/cat_contigs.nf

```diff
@@ -0,0 +1,46 @@
+process CAT_CONTIGS {
+    tag "${meta.id}-${db_name}"
+    label 'process_high'
+
+    conda "bioconda::cat=4.6 bioconda::diamond=2.0.6"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cat:5.2.3--hdfd78af_1' :
+        'quay.io/biocontainers/cat:5.2.3--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(assembly)
+    tuple val(db_name), path("database/*"), path("taxonomy/*")
+
+    output:
+    path("*.names.txt.gz")                    , emit: tax_classification
+    path("raw/*.ORF2LCA.txt.gz")              , emit: orf2lca
+    path("raw/*.predicted_proteins.faa.gz")   , emit: faa
+    path("raw/*.predicted_proteins.gff.gz")   , emit: gff
+    path("raw/*.log")                         , emit: log
+    path("raw/*.contig2classification.txt.gz"), emit: tax_classification_taxids
+    path "versions.yml"                       , emit: versions
+
+    script:
+    def official_taxonomy = params.cat_official_taxonomy ? "--only_official" : ""
+    """
+    CAT contigs -c "$assembly" -d database/ -t taxonomy/ -n 4 -o "${meta.id}" --top 11 --I_know_what_Im_doing
+    CAT add_names -i "${meta.id}.ORF2LCA.txt" -o "${meta.id}.ORF2LCA.names.txt" -t taxonomy/ ${official_taxonomy}
+    CAT add_names -i "${meta.id}.contig2classification.txt" -o "${meta.id}.contig2classification.names.txt" -t taxonomy/ ${official_taxonomy}
```

Comment:
I usually find it more readable with
```suggestion
    CAT add_names \\
        -i "${meta.id}.contig2classification.txt" \\
        -o "${meta.id}.contig2classification.names.txt" \\
        -t taxonomy/ ${official_taxonomy}
```
optional, might be personal preference only
Additionally, usually we use 
`def prefix = task.ext.prefix ?: "${meta.id}"`
below the `script:` line instead of using directly `${meta.id}`